packages/tools-node/fs.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./lib/fs");
```

Comment:
I would prefer if we just spell out the names, i.e.
```suggestion
module.exports = require("./lib/filesystem");
```

Reply:
for this one case, i disagree. fs is already established as the "file system" module name in the nodejs world.